.github/workflows/release-please.yml

```diff
@@ -0,0 +1,123 @@
+on:
+  push:
+    branches:
+      - release/testnet
+      - release/integrationnet
+      - release/mainnet
+
+name: Release please
```

Comment:
Maybe just `Release`? Since we do not use release-please anymore. Same with file name.